lua/orgmode/babel/tangle.lua

```diff
@@ -60,15 +61,51 @@ function Tangle:tangle()
       vim.fn.mkdir(path, 'p')
     end
 
+    local shebang = info.header_args[':shebang']
+    if shebang then
+      shebang = shebang:gsub('[\'"]', '')
+      table.insert(parsed_content, 1, shebang)
+      if filemode == nil then
+        filemode = 'o755'
+      end
+    end
+
+    local tangle_mode = info.header_args[':tangle-mode']
+    if tangle_mode then
+      filemode = tangle_mode:gsub('[\'"]', '')
+    end
+
+    if info.header_args[':mkdirp'] == 'yes' then
+      local path = vim.fn.fnamemodify(info.filename, ':h')
+      vim.fn.mkdir(path, 'p')
+    end
+
+    local shebang = info.header_args[':shebang']
+    if shebang then
+      shebang = shebang:gsub('[\'"]', '')
+      table.insert(parsed_content, 1, shebang)
+    end
```

Comment:
This logic is already done above on line 64. Is there a reason why it's needed twice?